### PR TITLE
Update simple agent references to app in quickstart

### PR DESCRIPTION
**Pull request by arjun-krishna1**
- Point to `simple-app` in quickstart
https://github.com/8090-inc/xrx-core/issues/16

### CODE REVIEW

File: docs/content/quickstart.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ git submodule update --init --recursive
 
 ## External Services Configuration
 
-xRx requires three external services: LLM, Text-to-Speech, and Speech-to-Text. Configure these services by setting the environment variables in the `.env` file inside the `simple-agent` application.
+xRx requires three external services: LLM, Text-to-Speech, and Speech-to-Text. Configure these services by copying the `env-example.txt` to a `.env` file and setting the environment variables inside the `simple-app` application.
```
Comment (arjun-krishna1) [inline docs/content/quickstart.md:97]: This was another thing that tripped me up for a second when setting this up.
There wasn't a .env or a .env.example file
But there are env-example.txt files with the right format
I've updated the instructions to make this explicit

Comment (sabbyanandan) [reply docs/content/quickstart.md:97]: Good change; thanks for fixing it.

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
@arjun-krishna1: This is a good change; however, there are a few other references to "Simple Agent" in the quickstart; it would be nice to clean then up, also.

Example: https://github.com/8090-inc/xrx-core/blob/main/docs/content/quickstart.md#how-to-run-the-simple-agent-application

**Comment by arjun-krishna1** [comment]
Hi @sabbyanandan, thanks for the review!
I think I have actually updated all references to "Simple Agent" to `simple-app`
For example the section you pointed out on main is now renamed to "How to run simple app" on this branch
https://github.com/8090-inc/xrx-core/blob/6ca2826a9b87c6434a46be52b77fe5f567df9c23/docs/content/quickstart.md#how-to-run-simple-app

**Review by sabbyanandan — APPROVED** [review]
:shipit:

**Comment by sabbyanandan** [comment]
Thanks, @arjun-krishna1!

**Pull request merged**